test-cases/upgrades/rolling-upgrade.yaml

```diff
@@ -0,0 +1,34 @@
+# TODO: need to qualify on GCE and AWS
```

Comment:
This has pending changes in PR from @amoskong.
We'll let the changes go in to the original files and after we merge both PRs, either you or @amoskong will change this one.

Reply:
I've rebased based on @amoskong changes